TrainingOOP-PHP001/listPoint.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace TrainingOOP1;
+
+require_once('Point.php');
+
+use TrainingOOP1\Point;
+
+class ListPoint {
+    public $points = [];
+
+    // Constructor
+    public function __construct() {
+        $arrX = [1, 3, 5, 7, 9];
+        $arrY = [2, 4, 6, 8, 10];
+
+        // Create a Point object and store it in the $points array
+        for ($ii = 0; $ii < count($arrX); $ii++) {
+            $this->points[$ii] = new Point($arrX[$ii], $arrY[$ii]);
+        }
+    }
+
+    // Calculate the distance between two points
+    public function distance($i, $j) {
```

Comment:
Chú ý format function comment:
```suggestion
    /**
     * Calculate the distance between two points
     * @param int $i First point index
     * @param int $j Second point index
     * @return int Distance of 2 points
     */
    public function distance($i, $j) {
```